odl/solvers/advanced/proximal_operators.py

```diff
@@ -375,6 +377,112 @@ def _call(self, x, out):
     return _ProximalConvConjL1
 
 
+# TODO: move notes to ODL doc
+def proximal_convexconjugate_kl(space, lam=1, g=None):
+    """Proximal operator factory of the convex conjugate of the KL divergence.
+
+    Function returning the proximal operator of the convex conjugate of the
+    functional F where F is the entropy-type Kullback-Leibler (KL) divergence
+
+        F(x) = sum_i (x - g + g ln(g) - g ln(pos(x)))_i + ind_P(x)
+
+    with x and g in X and g non-negative. The indicator function ind_P(x)
+    for the positive elements of x is used to restrict the domain of F such
+    that F is defined over whole X. The non-negativity thresholding pos is
+    used to define F in the real numbers.
+
+    The proximal operator of the convex conjugate, F_cc, of F is
+
+        F_cc(p) = sum_i (-g ln(pos(1_X - p))_i + ind_P(1_X - p)
+
+    where p is the variable dual to x, and 1_X is a vector in the space X with
+    all components set to 1.
+
+    The proximal operator of the convex conjugate of F is
+
+        prox_sigma[F_cc](x) = 1/2 (lam_X + x - sqrt((x - lam_X)^2 + 4 lam
+        sigma g)
```

Comment:
Regarding math explanations in docstrings: I think we should consider doing it in the NumPy/SciPy style, namely to give only a short explanation in the class docstring (hardly any math) and then use the `Notes` section to introduce the math. There one can go wild and use LaTeX commands and all that. See [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections), points 3. and 10.